### DISCUSSION TITLE
[BUG] Botones de creación de tareas (vista task) tienen distinto comportamiento

### DISCUSSION THREAD

**Discussion by jdurandpaz** [Cotalker/documentation#71] [4. Bug Reports] [answered]
Hola,

Mi consulta es si por algun razón en particular los botones (+) para crear tareas de la vista de task tienen distinto comportamiento, ya que por el momento mediante required survey solo he podido crear tareas con uno de ellos

**BOTON + EN VISTA FILTRADA**

![image](https://user-images.githubusercontent.com/50208672/114460167-6b7f0280-9baf-11eb-8fd5-920c01cd2296.png)

- Aparece un error

![image](https://user-images.githubusercontent.com/50208672/114460219-82255980-9baf-11eb-9dcf-1c113f56a00c.png)

- Y al ver las llamadas de red

error: "Error: Required survey not configured for task group initial state machine"

**BOTON EN VISTA SIN FILTRAR DENTRO DE UNA TASK**

![image](https://user-images.githubusercontent.com/50208672/114461683-0972cd00-9bb0-11eb-8dac-1511910b9cd5.png)

- Tarea se crea correctamente, en el proyecto que indique uno en el form y no necesariamente en el proyecto que uno selecciono por razones obvias, ya que en el form se completa cliente y proyecto

**BOTON SUPERIOR EN VISTA TASK**

- Se crea tarea pero no aparece required survey, asumo por ser una sub-máquina

![image](https://user-images.githubusercontent.com/50208672/114461098-c4e73180-9baf-11eb-8343-327960a78a75.png)

**Comment by guillevalin** [comment, marked as answer]
@jdurandpaz se levantó un ticket interno para corregir este incidente ya que se encontraron dos problemas:

1.- Required survey no es concordante respecto del botón superior versus los inferiores.
2.- Error al crear task con required survey (endpoint falla).

**Reply by jdurandpaz** [reply]
gracias guille, cree un ticket desde ops asi que para no confundir ideal si lo peudes borrar #2697
